wasm/src/program/transaction.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+  account::{Address, PrivateKey},
+  record::RecordPlaintext
+};
+use crate::{
+    Aleo,
+    Identifier,
+    Process,
+    Program,
+    ProvingKey,
+    TransactionNative
+};
+
+use std::str::FromStr;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct TransactionBuilder {}
+
+#[wasm_bindgen]
+impl TransactionBuilder {
+    /// Creates an execute transaction from a full proof of execution
+    pub fn build_transfer_full(
+        private_key: PrivateKey,
+        proving_key: ProvingKey,
+        address: Address,
+        amount: u64,
+        record: RecordPlaintext,
+    ) -> String {
+        console_error_panic_hook::set_once();
+
+        let process = Process::load_wasm().unwrap();
+        let credits_program = Program::credits().unwrap();
```

Comment:
In general we want to avoid using unwraps here. Usually we use anyhow's error handling https://docs.rs/anyhow/latest/anyhow/ which we transform to a generic Result enum here.

```suggestion
        let process = Process::load_wasm().map_err(|_| "Could not initialize wasm".to_string())?;
        let credits_program = Program::credits().map_err(|_| "Could not access credits program".to_string())?;
```